src/components/TagList/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Chip from '@/components/Chip';
+import style from './styles';
+
+interface tagsProps {
```

Comment:
p5;
인터페이스 or 타입이다보니 파스칼 케이스 어떨까요!?

```suggestion
interface TagsProps {
```